### ISSUE TITLE
Plugin incompatible with Matomo 3.5+, GeoIp2 now supported by core

### ISSUE THREAD

**Issue by sgiehl** [diabl0/piwik-geoip2#14]
With Matomo 3.5 we finally added support for Maxmind GeoIP2 databases to core.

Due to various changes in core this plugin is not compatible with Matomo 3.5 or later

@diabl0 thanks for providing this plugin and making GeoIP2 support available sooner!

**Comment by RAIMUNDOS54** [comment]
How does one set up the GeoIP2 in Matomo 3.6, for example? Just updated. Thanks.

**Comment by sgiehl** [comment]
You can simply setup and select GeoIP2 as Location Provider in Matomo backend. Like it used to be for GeoIP before.

**Comment by RAIMUNDOS54** [comment]
No. It’s not. The ones using the extension previously need to activate a new plugin in Matomo to have it working. Very bad thing. And this should also be specified in the documentation.



De: Stefan Giehl
Enviado:quinta-feira, 27 de setembro de 2018 04:34
Para: diabl0/piwik-geoip2
Cc:RAIMUNDOS54; Comment
Assunto: Re: [diabl0/piwik-geoip2] Plugin incompatible with Matomo 3.5+,GeoIp2 now supported by core (#14)

You can simply setup and select GeoIP2 as Location Provider in Matomo backend. Like it used to be for GeoIP before.
